app/src/main/java/edu/grinnell/appdev/grinnelldirectory/activities/DetailActivity.java

```diff
@@ -25,11 +42,17 @@
 
     public static final int ANIMATION_DURATION = 500;
 
-    Person p;
+    private Person p;
     boolean isImageZoomed;
-    Pair<Float, Float> screenDimens;
-    Pair<Float, Float> initialPicDimens;
-    Pair<Float, Float> zoomedPicTranslate;
+    private Pair<Float, Float> screenDimens;
+    private Pair<Float, Float> initialPicDimens;
+    private Pair<Float, Float> zoomedPicTranslate;
+
+    private String basePhoneNum = "641269";
```

Comment:
It doesn't really make a big difference, but add the `final` tag onto this to indicate that it's a constant.

Reply:
`zoomedPicTranslate` can't be final because it can't be initialized until after the activity is inflated.